### DISCUSSION TITLE
World.setBlock() does not work

### DISCUSSION THREAD

**Discussion by mazemuster7** [CrossCraft/CrossCraft-SurvivalTest#11] [General]
I'm trying to write a mod but using `World.setBlock(119, 53, 106, 1)` (I'm also using arguments correctly) returns an error in the console.
Error: ``error running function `onStart`': mods/dungeonpit/script.lua:2: attempt to call a nil value (field 'setBlock')``
Upon looking inside [Mod.cpp](https://github.com/CrossCraft/CrossCraft-SurvivalTest/blob/main/src/Modding/Mod.cpp), I found that line 173 references the function as World.setBlk(). Upon trying that instead, nothing happens.

**EDIT:** World.setBlk() does work. It was just that I wasn't landing on it. Either way, [MODDING.MD](https://github.com/CrossCraft/CrossCraft-SurvivalTest/blob/main/MODDING.MD) is incorrect.

**Comment by IridescentRose** [comment]
Updated
